packages/webdriverio/src/utils/getElementObject.ts

```diff
@@ -87,18 +88,18 @@ export const getElements = function getElements(selector, elemResponse, isReactE
                  * set element id with proper key so element can be passed into execute commands
                  */
                 if (this.isW3C) {
-                    client[ELEMENT_KEY] = elementId
+                    (client as any)[ELEMENT_KEY] = elementId
                 } else {
                     client.ELEMENT = elementId
                 }
             } else {
-                client.error = res
+                client.error = res as unknown as Error
```

Comment:
it would be better to type  `res` as `... | Error`, then you don't need the cast to `as unknown` first